c2corg_ui/templates/route/view.html

```diff
@@ -193,7 +193,8 @@ <h3 class="heading show-phone"><span translate>description</span></h3>
   <button class="btn btn-default scroll-top-btn" onclick="window.scrollTo(0, 0);">&nbsp;<span class="glyphicon glyphicon-menu-up"></span></button>
 </div>
 
-<div class="ng-hide">
+<div class="ng-hide" ng-init="associations = ${route.get('associations')}">
+  {{detailsCtrl.documentService.associations= associations}}
```

Comment:
Idem https://github.com/c2corg/v6_ui/pull/372/files#r65093095
Do we have other cases like these?